src/Symfony/Component/Validator/Constraints/UrlValidator.php

```diff
@@ -35,7 +35,7 @@ class UrlValidator extends ConstraintValidator
             )
             (:[0-9]+)?                              # a port (optional)
             (?:/ (?:[\pL\pN\-._\~!$&\'()*+,;=:@]|%%[0-9A-Fa-f]{2})* )*      # a path
-            (?:\? (?:[\pL\pN\-._\~!$&\'()*+,;=:@/?]|%%[0-9A-Fa-f]{2})* )?   # a query (optional)
+            (?:\? (?:[\pL\pN\-._\~!$&\'()*+,;\\\=:@/?]|%%[0-9A-Fa-f]{2})* )?   # a query (optional)
```

Comment:
`\\` should be enough, right?

Reply:
Didn't pass tests that way. I decided it was 1 escape for PHP, 1 escape for regex, and 1 for the actual slash.